_speakers/darren-hunt.md

```diff
@@ -16,14 +15,15 @@ twitter:
  - 
 # linkedin: 
 takeaways:
- - A greater insight into the value of User Testing
- - Tips on conducting effective User Testing
- - What to look for when recruiting participants for User Testing
+ - greater insight into the value of User Testing
+ - tips on conducting effective User Testing
+ - what to look for when recruiting participants for User Testing
 
 ---
 
 <p>Q&A session with an expert panel. Ask our panel any questions you may have about all areas of user experience, such as: Why should we user test? What is the best approach? What's the importance of hiring the right participants?
- 
+
```

Comment:
Need to add <p></p> tags - check your formatting by viewing his page. 

Suggest: 

<p>the panel so far: </p>
<ol>
<li> Paul Gooding .... </li>
<li>Darren Hunt ... </li>
....
</ol>